.github/workflows/demo-cd.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         with:
           images: |
             ghcr.io/pluralsh/flaky-service
+            ghcr.io/pluralsh/flaky-service-sidecar
```

Comment:
for the sidecar image, you can just use a busybox image and wire in a command that calls curl in a loop by running `/bin/bash -c "..."`